### PR TITLE
Updated PHP Version String for 8.0 and 8.1

### PR DESCRIPTION
**Pull request by hmennen90**
Ran tests in PHP 8.1 locally

### CODE REVIEW

File: composer.json [hunk 1]
```diff
@@ -16,7 +16,7 @@
         }
     ],
     "require": {
-		"php": ">=7.2.5",
+		"php": ">=7.2.5|^8.0|^8.1",
```
Comment (ilvalerione) [inline composer.json:19]: What's the issue with the expression ">=7.2"?

Comment (hmennen90) [reply composer.json:19]: It is the same Version String as in inspector-laravel package, wasn't it?

Comment (ilvalerione) [reply composer.json:19]: Yes, but it is not necessary. The expression ">=7.2" includes also the version 8.

Comment (hmennen90) [reply composer.json:19]: Ok - then i will close this pr

**Pull request closed without merging**